clang/include/clang/Format/Format.h

```diff
@@ -3146,6 +3146,45 @@ struct FormatStyle {
   /// \version 16
   IntegerLiteralSeparatorStyle IntegerLiteralSeparator;
 
+  /// Function-like declaration with keyworded parameters.
+  /// Lists possible keywords for a named macro-like function
```

Comment:
```suggestion
  /// Lists possible keywords for a named macro-like function.
```

Reply:
Lists possible keywords for a named function-like macro.